server/ssrComponents/imageModal.html

```diff
@@ -0,0 +1,8 @@
+<div id="image-modal" class="image-modal">
+    <button type="button" class="close" title="Minimize this image" aria-label="Minimize this image">
+      <!-- svgPlaceholder -->
```

Comment:
I think it would be fine to inline the SVGs here too, and switch on them using CSS.

Reply:
sounds good! I'll inline the svg

Reply:
also, this might have been a little confusing given the initial implementation - writing for posterity.

Only the minimize icon is used in the modal. The expand icon is used as an overlay on images before they're expanded, so they can't both be added to the modal.